packages/hash/frontend/src/shared/layout/layout-with-sidebar/workspace-switcher.tsx

```diff
@@ -67,15 +69,18 @@ export const WorkspaceSwitcher: FunctionComponent<
         subText: `@${authenticatedUser.shortname ?? "user"}`,
         avatarTitle: authenticatedUser.preferredName ?? "U",
       },
-      ...authenticatedUser.memberOf.map(({ orgAccountId, name, members }) => ({
+      ...authenticatedUser.memberOf.map(({ orgAccountId, name }) => ({
         key: orgAccountId,
         url: `/${orgAccountId}`,
         title: name,
-        subText: `${members.length} members`,
+        subText: `${
+          orgs.find((org) => org.orgAccountId === orgAccountId)?.members
+            .length ?? "Unknown number of "
```

Comment:
I think it would, however I've reverted some of the changes to this file because I spent time on the recursion stuff to reintroduce references to the org objects.